crates/pop-cli/src/commands/up/parachain.rs

```diff
@@ -26,6 +27,12 @@ pub(crate) struct ZombienetCommand {
 	/// The url of the git repository of a parachain to be used, with branch/release tag specified as #fragment (e.g. 'https://github.com/org/repository#tag'). A specific binary name can also be optionally specified via query string parameter (e.g. 'https://github.com/org/repository?binaryname#tag'), defaulting to the name of the repository when not specified.
 	#[arg(short, long)]
 	parachain: Option<Vec<String>>,
+	/// The path to a script to be run after the network has been set up.
+	#[clap(name = "init", short = 'i', long)]
+	setup_script: Option<String>,
```

Comment:
Something more generic like `command` would be better as it could be anything added here instead of a script - e.g. `echo "hi!"`

The separate args can then be removed so it becomes `-i "echo hi!"`

Reply:
Updated.